node/node.go

```diff
@@ -407,76 +407,33 @@ func (node *Node) RemovePeersHandler() {
 	}
 }
 
-func (node *Node) initWalletNodeConfiguration() (service.NodeConfig, chan p2p.Peer) {
+// isBeacon = true if the node is beacon node
+// isClient = true if the node light client(txgen,wallet)
+func (node *Node) initNodeConfiguration(isBeacon bool, isClient bool) (service.NodeConfig, chan p2p.Peer) {
```

Comment:
we can actually use the global nodeconfig now, instead of passing the parameters here.

Reply:
Currently we are using NodeConfig and ConfigType, let's remove NodeConfig in another pr